scss/_utilities/_clearfix.scss

```diff
@@ -0,0 +1,19 @@
+// ==========================================================================
+// Clearfix
+// ==========================================================================
+
+
+@mixin clearfix() {
+	&:before,
+	&:after {
+		content: "";
+		display: table;
+	}
+	&:after {
+		clear: both;
+	}
+}
+
+.clearfix {
```

Comment:
We can leave it in for name (the name), and if we decided to have different naming conventions for #4, then we can change it.

Also might as well leave the clearfix mixin there for now, since in some cases it could definitely be handy to have.